src/App.vue

```diff
@@ -377,6 +377,14 @@ export default {
 				return
 			}
 
+			if (from.name === 'conversation') {
```

Comment:
It's also triggered when hash/query changed (click on reply quote to reproduce).
That was the vital check in LeftSidebar before executing. Ideally, that should be in a router guard, but for now this should suffice:

```suggestion
			if (from.name === 'conversation' && from.params.token !== to.params.token) {
```

And same for L391:
```js
			if (to.name === 'conversation' && from.params.token !== to.params.token) {
```